terraform/shared/shared.tf.json.template.py

```diff
@@ -40,7 +40,12 @@ def conformance_pack(name: str) -> str:
     # https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-cis-controls.html#securityhub-cis-controls-3.1
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Average',
-                    filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}'),
+                    filter_pattern='{($.errorCode = "*UnauthorizedOperation" || $.errorCode = "AccessDenied*") && '
+                                   '($.eventSource != "s3.amazonaws.com" && '
```

Comment:
It's really difficult to parse the condition the way you've wrapped it. Especially, the way you've "hid" the nesting by ignoring the parentheses. When an expression in a string literal becomes more complicated, we usually resort to using `str.join` as in `'&&'.join(...).  If that doesn't work, and if we wrap at the site of a binary operator, we wrap before the operator so that it appears at the beginning of the line. 

The additional conditions needs a brief comment motivating them. Remember not to explain the "how" since that it obvious, but the "why".